ansible/deploy_backend_ultimate.yml

```
@@ -0,0 +1,197 @@
+          Environment=JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
+          Environment=BASE_URL={{ backend_url }}
+          Environment=PUBLIC_URL=
+          Environment=STATIC_FOLDER=/frontend
+          Environment=UPLOAD_PATH={{ backend_dest }}/uploads
+          Environment=URL_LOGS=/var/log/teachua
+          
+          # Database configuration
+          Environment=DEV2_DATASOURCE_URL=jdbc:postgresql://34.173.19.35:5432/teachua
+          Environment=DEV2_DATASOURCE_USER=teachua_user
+          Environment=DEV2_DATASOURCE_PASSWORD=securepassword123
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)